sample/source/shared/GeneratedAssets.cpp

```diff
@@ -156,40 +210,47 @@ const auto g_diffusePaths = std::array{
     std::string{path::diffuse_yellow},
     std::string{path::effect_linear_hatch},
     std::string{path::effect_noise},
-};
-
-const auto g_normalPaths = std::array{
     std::string{path::normal_cave},
     std::string{path::normal_guy},
     std::string{path::normal_ogre},
     std::string{path::normal_skeleton},
 };
 
-nc::asset::TextureView diffuse_cave{};
-nc::asset::TextureView diffuse_guy{};
-nc::asset::TextureView diffuse_ogre{};
-nc::asset::TextureView diffuse_skeleton{};
-nc::asset::TextureView diffuse_blue{};
-nc::asset::TextureView diffuse_green{};
-nc::asset::TextureView diffuse_orange{};
-nc::asset::TextureView diffuse_purple{};
-nc::asset::TextureView diffuse_red{};
-nc::asset::TextureView diffuse_teal{};
-nc::asset::TextureView diffuse_yellow{};
-nc::asset::TextureView effect_linear_hatch{};
-nc::asset::TextureView effect_noise{};
-nc::asset::TextureView normal_cave{};
-nc::asset::TextureView normal_guy{};
-nc::asset::TextureView normal_ogre{};
-nc::asset::TextureView normal_skeleton{};
+const auto g_subtypes = std::array{
+    nc::asset::AssetSubtype::DiffuseTexture,
```

Comment:
No need to change now, but upon further thought 'Diffuse' isn't the right name for these. The difference between these and the normals is the texture format. I would actually call these either `ColorTexture` or `RgbaTexture`

Reply:
No better time to change than now.